[E 2021-08-08 22:02:27,692 /Users/zhongrenguang/opt/anaconda3/lib/python3.7/site-packages/shioaji/utils.py:42:raise_resp_error] {'status': {'status_code': 400}, 'response': {'token': '', 'accounts': [], 'detail': 'ORA-20003: customer_not_exist\nORA-06512: at "STSTX.SGTP_LOGIN_G", line 395\nORA-06512: at line 1', 'ip_address': '219.68.49.137', 'client_name': ''}}
[E 2021-08-08 22:03:24,599 /Users/zhongrenguang/opt/anaconda3/lib/python3.7/site-packages/shioaji/utils.py:42:raise_resp_error] {'status': {'status_code': 400}, 'response': {'token': '', 'accounts': [], 'detail': 'ORA-20003: customer_not_exist\nORA-06512: at "STSTX.SGTP_LOGIN_G", line 395\nORA-06512: at line 1', 'ip_address': '219.68.49.137', 'client_name': ''}}
[E 2021-08-08 22:03:34,179 /Users/zhongrenguang/opt/anaconda3/lib/python3.7/site-packages/shioaji/utils.py:42:raise_resp_error] {'status': {'status_code': 400}, 'response': {'token': '', 'accounts': [], 'detail': 'ORA-20003: customer_not_exist\nORA-06512: at "STSTX.SGTP_LOGIN_G", line 395\nORA-06512: at line 1', 'ip_address': '219.68.49.137', 'client_name': ''}}
[E 2021-08-08 22:03:40,471 /Users/zhongrenguang/opt/anaconda3/lib/python3.7/site-packages/shioaji/utils.py:42:raise_resp_error] {'status': {'status_code': 400}, 'response': {'token': '', 'accounts': [], 'detail': 'ORA-20003: customer_not_exist\nORA-06512: at "STSTX.SGTP_LOGIN_G", line 395\nORA-06512: at line 1', 'ip_address': '219.68.49.137', 'client_name': ''}}
[E 2021-08-08 22:03:44,644 /Users/zhongrenguang/opt/anaconda3/lib/python3.7/site-packages/shioaji/utils.py:42:raise_resp_error] {'status': {'status_code': 400}, 'response': {'token': '', 'accounts': [], 'detail': 'ORA-20003: customer_not_exist\nORA-06512: at "STSTX.SGTP_LOGIN_G", line 395\nORA-06512: at line 1', 'ip_address': '219.68.49.137', 'client_name': ''}}
[E 2021-08-08 22:03:47,837 /Users/zhongrenguang/opt/anaconda3/lib/python3.7/site-packages/shioaji/utils.py:42:raise_resp_error] {'status': {'status_code': 400}, 'response': {'token': '', 'accounts': [], 'detail': 'ORA-20003: customer_not_exist\nORA-06512: at "STSTX.SGTP_LOGIN_G", line 395\nORA-06512: at line 1', 'ip_address': '219.68.49.137', 'client_name': ''}}
[E 2021-08-08 22:03:51,698 /Users/zhongrenguang/opt/anaconda3/lib/python3.7/site-packages/shioaji/utils.py:42:raise_resp_error] {'status': {'status_code': 400}, 'response': {'token': '', 'accounts': [], 'detail': 'ORA-20003: customer_not_exist\nORA-06512: at "STSTX.SGTP_LOGIN_G", line 395\nORA-06512: at line 1', 'ip_address': '219.68.49.137', 'client_name': ''}}
[E 2021-08-08 22:03:54,834 /Users/zhongrenguang/opt/anaconda3/lib/python3.7/site-packages/shioaji/utils.py:42:raise_resp_error] {'status': {'status_code': 400}, 'response': {'token': '', 'accounts': [], 'detail': 'ORA-20003: customer_not_exist\nORA-06512: at "STSTX.SGTP_LOGIN_G", line 395\nORA-06512: at line 1', 'ip_address': '219.68.49.137', 'client_name': ''}}
[E 2021-08-08 22:06:16,673 /Users/zhongrenguang/opt/anaconda3/lib/python3.7/site-packages/shioaji/utils.py:42:raise_resp_error] {'status': {'status_code': 400}, 'response': {'token': '', 'accounts': [], 'detail': 'ORA-20003: customer_not_exist\nORA-06512: at "STSTX.SGTP_LOGIN_G", line 395\nORA-06512: at line 1', 'ip_address': '219.68.49.137', 'client_name': ''}}
